opentracing-noop/src/main/java/io/opentracing/noop/NoopTracerFactory.java

```diff
@@ -13,12 +13,18 @@
  */
 package io.opentracing.noop;
 
+import io.opentracing.ActiveSpanSource;
+
 public final class NoopTracerFactory {
     
     public static NoopTracer create() {
         return NoopTracerImpl.INSTANCE;
     }
 
+    public static NoopTracer create(ActiveSpanSource spanSource) {
+        return new NoopTracerImpl(spanSource);
```

Comment:
True - but it should only result in a single instance as it will generally only be called once to initialise the tracer.